RFC-0013-view-inplace-handling-on-opaque-tensors.md

```diff
@@ -0,0 +1,98 @@
+## Summary
+
+In the ideal end state, dispatcher should be able to provide an opt-in
+kernel to functionalize view & inplace ops. Currently we have a similar
+infrastructure in pytorch/xla, but that’s only available on the lazy
+traced graph. To make this elimination pass available to FX and Mobile
+GPU like other backends that only support opaque tensor storage, this
+pass should be lifted up to dispatcher level.
```

Comment:
@ailzhang Ed tells me you have a prototype somewhere. Do you have a link to it and is it usable? I'm interested in trying it out on some functorch-related things. Thanks!

Reply:
(found at https://github.com/ailzhang/pytorch/commit/63f070e8185428e4c770bf7f5defd61591450435 !)